scripts/miscContracts.ts

```diff
@@ -98,7 +98,7 @@ export async function getMiscAddresses() {
 		data: {
 			name,
 			protocol,
-			logoURI: undefined,
+			logoURI: 'images/contracts/unknown.png',
```

Comment:
I feel like `undefined` is the correct choice here?  Deciding how to present a contract without a logo should be up to the application using this metadata, not the metada.  The styling may differ between apps (e.g., rounded corners, color theme, etc.)